demosplan/DemosPlanCoreBundle/Resources/config/permissions.yml

```diff
@@ -2584,6 +2584,12 @@ field_statement_municipality:
     label: 'Show field "municipality" (Gemeinde) and save its value'
     loginRequired: false
     parent: field_statement
+field_statement_paragraph:
```

Comment:
We should be very sure that there is not already a permission for that. 
In many other cases the option to link a paragraph is correctly hidden for this project, that's why I guess there is already a permission which we should use.

Reply:
Maybe `field_procedure_paragraphs`?

Reply:
for refs:
`feature_documents_new_statement` it is

Reply:
Changed in [cf93172](https://github.com/demos-europe/demosplan-core/pull/628/commits/cf931724ee7820b5a87f4e6e74e549c0e2324f80)